web/satellite/src/utils/apolloManager.ts

```diff
@@ -9,7 +9,7 @@ import { getToken } from '@/utils/tokenManager';
 
 // Satellite url
 const satelliteUrl = new HttpLink({
-    uri: '/api/graphql/v0',
+    uri: 'http://localhost:10002/api/graphql/v0',
```

Comment:
This should not be hardcoded, right?

Reply:
Yes, sure. This is for test purposes only and will be reverted before Merging PR.